packages/cli-exec/src/exec.js

```diff
@@ -0,0 +1,108 @@
+import command from '@percy/cli-command';
+import * as common from './common';
+
+import start from './start';
+import stop from './stop';
+import ping from './ping';
+
+export const exec = command('exec', {
+  description: 'Start and stop Percy around a supplied command',
+  usage: '[options] -- <command>',
+  commands: [start, stop, ping],
+
+  flags: [...common.flags, {
+    name: 'parallel',
+    description: 'Marks the build as one of many parallel builds',
+    parse: () => !!(process.env.PERCY_PARALLEL_TOTAL ||= '-1')
+  }, {
+    name: 'partial',
+    description: 'Marks the build as a partial build',
+    parse: () => !!(process.env.PERCY_PARTIAL_BUILD ||= '1')
+  }],
+
+  examples: [
+    '$0 -- echo "percy is running around this echo command"',
+    '$0 -- yarn test'
+  ],
+
+  loose: [
+    'Warning: Missing command separator (--),',
+    'some command options may not work as expected'
+  ].join(' '),
```

Comment:
I hope this helps catch some weird stuff happening, like yarn stripping forwarding args lol